spyder/plugins/debugger/widgets/breakpoint_table_view.py

```diff
@@ -14,30 +14,31 @@
 # pylint: disable=R0911
 # pylint: disable=R0201
 
-# Standard library imports
-import sys
 
 # Third party imports
 from qtpy import PYQT5
 from qtpy.compat import to_qvariant
 from qtpy.QtCore import QAbstractTableModel, QModelIndex, Qt, Signal
-from qtpy.QtWidgets import QItemDelegate, QTableView, QVBoxLayout
+from qtpy.QtWidgets import QItemDelegate, QTableView
 
 # Local imports
-from spyder.api.translations import _
-from spyder.api.widgets.main_widget import (PluginMainWidgetMenus,
-                                            PluginMainWidget)
+from spyder.api.translations import get_translation
```

Comment:
```suggestion
from spyder.api.translations import _
```

Some months ago we found that the old way of importing translations was creating unnecessary objects all over the place, which consumed more memory. So, we prefer now to simply import the `_` function from the translations API.